e2e/validate/validation.go

```diff
@@ -117,12 +118,23 @@ func FindPod(cluster types.Cluster, namespace, labelSelector, podIdentifier stri
 		labelSelector, podIdentifier, namespace, cluster.Name)
 }
 
+// TestConfig is a wrapper function which performs validation checks on
+// the test environment configurations with the DR resources on the clusters.
+// Returns an error if any validation fails.
+func TestConfig(env *types.Env, config *types.Config, log *zap.SugaredLogger) error {
+	if err := clustersInDRPolicy(env, config, log); err != nil {
+		return err
+	}
+
+	return nil
+}
+
 // ValidateClustersInDRPolicy checks if configured clusters match the configured
 // drpolicy. Returns an error if cluster names are not the same as drpolicy
 // drclusters. The reason for a failure may be wrong cluster name or wrong
 // drpolicy.
-func ValidateClustersInDRPolicy(env *types.Env, config *types.Config, log *zap.SugaredLogger) error {
-	drpolicy, err := GetDRPolicy(env.Hub, config.DRPolicy)
+func clustersInDRPolicy(env *types.Env, config *types.Config, log *zap.SugaredLogger) error {
+	drpolicy, err := util.GetDRPolicy(env.Hub, config.DRPolicy)
 	if err != nil {
 		return err
```

Comment:
Lets build a good error here. We know the context (validating test config) and we know what is the issue. Returning good error here will help both dr tests and ramenctl test command.

Reply:
added